app.js

```diff
@@ -11,6 +11,7 @@ const {
 app.get("/api/topics", getTopics);
 app.get("/api", getEndpoints);
 
+app.get("/api/articles", getArticles);
 app.get("/api/articles/:article_id", getArticles);
```

Comment:
We shouldn't be re-using controllers. `getArticles` should only be the controller for `api/articles`. You should have a separate controller for `/api/articles/:article_id`, preferably called something like `getSingleArticle` 👍